From ff014657939f30c0052a1d2d66d87d13833c5255 Mon Sep 17 00:00:00 2001
From: Joachim Schmitz <jojo@schmitz-digital.de>
Date: Tue, 18 Sep 2018 14:42:43 +0200
Subject: [PATCH] changes needed for MuseScore to build with Qt 5.12 (Alpha)

One needed for MSVC only (and only since Alpha, not with the previous
preview), the other needed for MSVC and MinGW.
Neither seems to cause issues with Qt 5.9 (like AppVeyor and Travis CI
prove)
---
 mscore/editstringdata.cpp | 4 ++--
 mscore/prefsdialog.ui     | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/mscore/editstringdata.cpp b/mscore/editstringdata.cpp
index 21a621bac0..a86207e5db 100644
--- a/mscore/editstringdata.cpp
+++ b/mscore/editstringdata.cpp
@@ -52,7 +52,7 @@ EditStringData::EditStringData(QWidget *parent, QList<instrString> * strings, in
                   strg = (*_strings)[numOfStrings - i - 1];
                   _stringsLoc.append(strg);
                   QTableWidgetItem *newCheck = new QTableWidgetItem();
-                  newCheck->setFlags(Qt::ItemFlag(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled));
+                  newCheck->setFlags(Qt::ItemFlags(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled));
                   newCheck->setCheckState(strg.open ? Qt::Checked : Qt::Unchecked);
                   stringList->setItem(i, 0, newCheck);
                   QTableWidgetItem *newPitch = new QTableWidgetItem(midiCodeToStr(strg.pitch));
@@ -168,7 +168,7 @@ void EditStringData::newStringClicked()
             _stringsLoc.insert(i, strg);
             stringList->insertRow(i);
             QTableWidgetItem *newCheck = new QTableWidgetItem();
-            newCheck->setFlags(Qt::ItemFlag(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled));
+            newCheck->setFlags(Qt::ItemFlags(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled));
             newCheck->setCheckState(strg.open ? Qt::Checked : Qt::Unchecked);
             stringList->setItem(i, 0, newCheck);
             QTableWidgetItem *newPitch = new QTableWidgetItem(midiCodeToStr(strg.pitch));
diff --git a/mscore/prefsdialog.ui b/mscore/prefsdialog.ui
index 5dd493fdc7..21af750846 100644
--- a/mscore/prefsdialog.ui
+++ b/mscore/prefsdialog.ui
@@ -4276,7 +4276,6 @@ Adjusting latency can help synchronize your MIDI hardware with MuseScore's inter
    </item>
   </layout>
  </widget>
- <pixmapfunction>getPixmap</pixmapfunction>
  <customwidgets>
   <customwidget>
    <class>Awl::ColorLabel</class>
